GitHubAction/GitHubAction.domain/Presenters/IGithubPresenter.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GitHubAction.Domain.Presenters;
+
+public interface IGithubPresenter
```

Comment:
Not the expert but I think a presenter should not be defined in the Domain layer